README.md

```diff
@@ -1,131 +1,95 @@
+The Molecular Informatics Vienna platform consists of three teams:
+1. [Christian-Doppler Laboratory for Molecular Informatics in the Bioscience team](https://cdlab-mib.univie.ac.at/)
+2. [Cheminfo team](https://cheminfo.univie.ac.at/home/)
+3. [comp3d team](https://comp3d.univie.ac.at/)
 
-## General Outline and Minimum Requirements
+This guide is designed to help everyone set up a molecular informatics project in Python quickly and easily, with a focus on creating reusable packages to maximize collaboration and efficiency.
 
-This page is split into **two** parts. One part is about general good-practices, wheras the second part is about the specifics of the CD-Lab Molecular Informatics in the Biosciences (CD-Lab MIB) platform.
+## Table of Contents
 
-1. Read through all of the documentation to get a grasp of general good-practices
-2. Read through **minimum** requirements that need to be met working with our collaborators in the CD-Lab Molecular Informatics in the Biosciences - see the [Minimum requirements working on code in the CD-Lab MIB:](/README.md#minimum-requirements-working-on-code-in-the-cd-lab-mib). 
+### Essential Practices (For Everyone)
+1. [Project Structure](./PROJECT_STRUCTURE.md)
+2. [Documentation, Code Style, and Linting](./DOCUMENTATION.md)
+3. [Version Control Basics](./VERSION_CONTROL.md)
+4. [Testing Fundamentals](./TEST)
+5. [Continues Integration (CI)](./CI.md)
+5. [IDE Setup for Beginners](./IDE.md)
 
-# Administrative
+### Advanced Topics (For Deeper Exploration)
+7. [Packaging and Distribution](./PACKAGING.md)
+8. [Advanced Continuous Integration (CI)](./ADVANCED_CI_CD.md)
+9. [GitHub Actions](./ADVANCED_CI_CD.md##Best_Practices_for_CI/CD)
+10. [Licensing](./LICENCING.md)
+11. [Machine Learning Resources](./ML.md)
 
-### New to the team?
+## Getting Started
+
+If you're new to the group or to coding in general, we recommend starting with the Essential Practices. These will help you set up a solid foundation for writing good Python code.
 
-First of all: `Welcome to the Molecular Informatics platform!`
+1. Begin by setting up your [Integrated Development Environment (IDE)](./IDE.md).
+2. Learn about proper [Project Structure](./PROJECT_STRUCTURE.md).
+3. Familiarize yourself with our [Documentation and Linting](./DOCUMENTATION.md) guidelines.
+4. Understand the basics of [Version Control](./VERSION_CONTROL.md).
+5. Learn about [Testing](./TEST.md) your code.
 
-We are glad to have you on-board :)
+For those looking to dive deeper or contribute to more complex projects, check out the Advanced Topics section.
 
-There are a few practices we consider worth following.
-1. <b>Communications:</b>
+## Minimum Requirements for CD-Lab MIB
```

Comment:
For minimum requirements we would like to suggest the following:

When working on code in the CD-Lab Molecular Informatics in the Biosciences, please ensure you follow these minimum requirements. A [template](Link_to_template) is provided to ensure all of these points. It should be copied to your project and used there.
1. Use the specified linters isort, black and pylint.
2. Use the specified documentation check with pydocstyle
3. Make sure the test coverage is above 80%.
4. Distribute your library as a package. For details see the [packaging](./PACKAGING.md) section

For further reading see also the sections [CI/CD Guidelines](./CI.md), [advanced](./ADVANCED_CI_CD.md), [linting](./LINTING.md), [good documentation](./DOCUMENTATION.md).

@spudlig we are of course open for better suggestions

Reply:
I adjusted the min requirements